configs/mesh_thread.json

```diff
@@ -11,20 +11,24 @@
         "enable-led-control-example": true,
         "LED": "NC",
         "BUTTON": "NC"
-    },  
+    },
+    "macros": ["MBEDTLS_USER_CONFIG_FILE=\"mbedtls_config.h\""],
```

Comment:
Are you sure that this builds on all the platforms that we are targeting in our CI?